doc/source/computation.rst

```diff
@@ -539,12 +549,12 @@ can even be omitted:
 .. ipython:: python
 
    covs = df[['B','C','D']].rolling(window=50).cov(df[['A','B','C']], pairwise=True)
-   covs[df.index[-50]]
+   covs.unstack(-1).iloc[-50]
```

Comment:
could also do a more explicit one like in the whatsnew example: `df.rolling(12).corr().loc['2016-04-07']` instead of the `df.index[-50]`